plugins/polio/js/src/domains/Chronogram/ChronogramTemplateTask/Table/useChronogramTemplateTaskTableColumns.tsx

```diff
@@ -29,6 +29,7 @@ export const useChronogramTemplateTaskTableColumns = (
                 id: 'description',
                 accessor: 'description',
                 sortable: false,
+                width: 800,
```

Comment:
Shouldn't we use an adaptive size here rather than a fixed size? Using a fixed size might behave incorrectly on smaller screen

Reply:
I double checked this morning and I can confirm that the layout is still adaptative, even though we're using a fixed size… 

I don't really know why but I guess it could be related to either CSS table layout or some responsive attributes.

In Safari:

![width_safari](https://github.com/user-attachments/assets/8cdfb0b9-5266-4b86-bffb-56267e29e801)

In Chrome:

![width_chrome](https://github.com/user-attachments/assets/b6029b00-5263-488e-b4e6-c5ecfe8fb66f)